packages/babel-preset-default/bin/index.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+require( 'core-js-builder' )( {
+	modules: [ 'es', 'web' ],
+	exclude: [ 'es.promise' ],
```

Comment:
Why do we have to explicitly exclude this, can't it be automatically handled by the browserlist config?

Reply:
This is because `core-js` replaces some native implementations of features with polyfills, if it deems that they have bugs that need to be fixed. The issue is that it's extremely conservative in this assessment, and it ends up replacing perfectly good implementations because of edge cases that never really come up.

Promises are a good example of this. As explained in their README:

> But sometimes `core-js` feature detection could be too strict for your case. For example, `Promise` constructor requires the support of unhandled rejection tracking and `@@species`.

These are rarely used features that shouldn't disqualify a native implementation.

Reply:
We may want to add an inline comment to explain that

Reply:
Right, I will copy the comment from @sgomes and reference this PR in code 👍🏻

Reply:
Added a comment in c385142.